libandroid-navigation-ui/build.gradle

```diff
@@ -1,4 +1,5 @@
 apply plugin: 'com.android.library'
+apply plugin: 'com.jaredsburrows.license'
```

Comment:
Noticed that licenses are generated only for `libandroid-navigation-ui` (we're not including the plugin in `libandroid-navigation`). Does the `license` plugin handle transitive dependencies nicely? Wondering if we should also generate the licenses for `libandroid-navigation` and have them somehow split by SDK (core / UI) in the `LICENSE` file? What do you think?

Reply:
That's a good idea. Its just that we may end up repeating the licenses since, both ui and core share common licenses.

Reply:
We could add a couple of headers like `Navigation SDK` and `Navigation UI SDK` and dump all of the licenses separately. Although we'd have duplicated licenses because UI depends on core, it may be useful for those not using `libandroid-navigation-ui`.